test/models/report_test.rb

```diff
@@ -3,7 +3,19 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '#editable?' do
+    target_user = User.new
+    non_target_user = User.new
+    report = target_user.reports.new
+
+    assert_equal true, report.editable?(target_user)
+    assert_equal false, report.editable?(non_target_user)
+  end
+
+  test '#created_on' do
+    today = Time.zone.today
+    @user = users(:alice)
+    report = @user.reports.create(title: 'test', content: 'This is test')
```

Comment:
```suggestion
    report = @user.reports.create!(title: 'test', content: 'This is test')
```

がお勧めです。

https://qiita.com/jnchito/items/6beb97e73934eb3e4dce#create%E3%81%A7%E3%81%AF%E3%81%AA%E3%81%8Fcreate%E3%82%92%E4%BD%BF%E3%81%86%E7%90%86%E7%94%B1

Reply:
`create!`に修正しました。